app/views/directives/osc-image-summary.html

```diff
@@ -4,10 +4,14 @@
   <div ng-show="resource | annotation:'provider'">Provider: {{ resource | annotation:'provider' }}</div>
   <div ng-show="resource.metadata.namespace">Namespace: {{ resource.metadata.namespace }}</div>
 </div>
-<div class="resource-description"
+<div class="resource-description pad-bottom-none"
      ng-bind-html="resource | description | linky"
```

Comment:
Yeah I don't think we need the sample repo showing up in both places. I really just want to link out to any relevant docs in the description. If anything, underneath the sample repo we could either embed the readme.md or link out to the readme, or mention something to users that they should look at the readme for the sample repo if they use it. For some reason this never seems to be common sense.